lib/ansible/modules/remote_management/redfish/redfish.py

```diff
@@ -0,0 +1,331 @@
+  command:
+    required: true
+    description:
+      - Command to execute on server
+  baseuri:
+    required: true
+    description:
+      - Base URI of OOB controller
+  user:
+    required: true
+    description:
+      - User for authentication with OOB controller
+  password:
+    required: true
+    description:
+      - Password for authentication with OOB controller
+  userid:
+    required: false
+    description:
+      - ID of user to add/delete/modify
+  username:
+    required: false
+    description:
+      - name of user to add/delete/modify
+  userpswd:
+    required: false
+    description:
+      - password of user to add/delete/modify
+  userrole:
+    required: false
+    description:
+      - role of user to add/delete/modify
+  bootdevice:
+    required: false
+    description:
+      - bootdevice when setting boot configuration
+  bios_attr_name:
+    required: false
+    description:
+      - name of BIOS attribute to update
+  bios_attr_value:
+    required: false
+    description:
+      - value of BIOS attribute to update to
+  mgr_attr_name:
+    required: false
+    description:
+      - name of Manager attribute to update
+  mgr_attr_value:
+    required: false
+    description:
+      - value of Manager attribute to update to
+
+author: "Jose Delarosa (github: jose-delarosa)"
+'''
+
+EXAMPLES = '''
+  - name: Getting system inventory
+    local_action: >
+       redfish category=Inventory command=GetSystemInventory baseuri={{baseuri}}
+       user={{user}} password={{password}}
+
+  - name: Get CPU Inventory
+    local_action: >
+       redfish category=Inventory command=GetCpuInventory baseuri={{baseuri}}
+       user={{user}} password={{password}}
+
+  - name: Enable PXE Boot for NIC1
+    local_action: >
+       redfish category=System command=SetBiosAttributes
+       user={{user}} password={{password}} baseuri={{baseuri}}
+       bios_attr_name=PxeDev1EnDis bios_attr_value=Enabled
+
+  - name: Set one-time boot device to {{bootdevice}}
+    local_action: >
+       redfish category=System command=SetOneTimeBoot baseuri={{baseuri}}
+       user={{user}} password={{password}} bootdevice={{bootdevice}}
+
+  - name: Turn system power on
+    local_action: >
+       redfish category=System command=PowerOn baseuri={{baseuri}}
+       user={{user}} password={{password}}
+
+  - name: Add user
+    local_action: >
+       redfish category=Accounts command=AddUser baseuri={{baseuri}}
+       user={{user}} password={{password}} userid={{userid}}
+       username={{username}} userpswd={{userpswd}} userrole={{userrole}}
+
+  - name: Enable user
+    local_action: >
+       redfish category=Accounts command=EnableUser baseuri={{baseuri}}
+       user={{user}} password={{password}} userid={{userid}}
+
+  - name: Get BIOS attributes
+    local_action: >
+       redfish category=System command=GetBiosAttributes baseuri={{baseuri}}
+       user={{user}} password={{password}}
+'''
+
+RETURN = '''
+changed:
```

Comment:
We don't usually add `changed` or `invocation` in `RETURN` as all modules return them